package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "gatsby": "^2.1.18",
-    "gatsby-image": "^2.0.35",
+    "gatsby-image": "^2.0.38",
```

Comment:
Why the update?

Reply:
Why not?

Reply:
Are we updating stuff just because now?

I have nothing against it tbh, just wanted to know if there was any issue this solved. Carry on

Reply:
During my adventure with the nested `noscript` issue I tried to update this just to see if something would be magically fixed. This was useless however. Should I revert it?

Reply:
Nope, it's good, just wanted to know why.